### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by d4nQw3rty**
This project name is Portfolio and this is a second branch where I will set up the header for the mobile version of the web I will develop.
In this branch, I have:
-Created the header section in HTML
-Given Styles to the header section and elements navigation bar, and headline.
-Made sure there are no linter errors.
-Used GitHub flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,38 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (HAMMAS-SALEEM) [inline README.md:2]: ### Your Summary
summary

### Descriptive Summary Example :heavy_check_mark:
hammas-summary


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

Comment (HAMMAS-SALEEM) [inline README.md:2]: ### Your Summary
summary

### Descriptive Summary Example :heavy_check_mark:
hammas-summary


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle: :traffic_light: :rotating_light:

## Hey @d4nQw3rty :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (d4nQw3rty) [reply README.md:2]: @HAMMAS-SALEEM
Kindly would you consider this an acceptable Summary?  
image

**Comment by d4nQw3rty** [comment]
This

**Comment by d4nQw3rty** [comment]
Hi @HAMMAS-SALEEM I have made a huge mistake a was trying to make some change in my project by making a meaningful summary as you request, and I realize that the name of my primary branch must be main not master, and my pull request must be feature branch, then I change the name of both but now my pull request is closed  look information in the technical question in the dash boar and found this [https://questions.microverse.org/t/pull-request-closed-and-wont-reopen/2530](url) can you please help me!!

**Pull request closed without merging**